person_signup.md

```diff
@@ -132,11 +132,11 @@ OSDI-API-Token:[your api key here]
         "gender": "Male"
     },
     "add_tags": [
-        "volunteer",
-        "donor"
+        "d91b4b2e-ae0e-4cd3-9ed7-d0ec501b0bc3",
```

Comment:
Have we decided definitively not to use full resource paths here, e.g. "https://www.osdisamplesystem.org/api/v1/tags/d91b4b2e-ae0e-4cd3-9ed7-d0ec501b0bc3"?  Or is this still up for discussion?